utils/zerovec/src/ule/plain.rs

```diff
@@ -230,6 +271,44 @@ impl AsULE for i8 {
 // EqULE is true because i8 is its own ULE.
 unsafe impl EqULE for i8 {}
 
+// Safety (based on the safety checklist on the ULE trait):
+//  1. NonZeroI8 does not include any uninitialized or padding bytes.
+//  2. NonZeroI8 is aligned to 1 byte.
+//  3. The impl of validate_byte_slice() returns an error if any byte is not valid (0x00).
+//  4. The impl of validate_byte_slice() returns an error if there are leftover bytes (never).
+//  5. The other ULE methods use the default impl.
+//  6. NonZeroI8 byte equality is semantic equality
+unsafe impl ULE for NonZeroI8 {
```

Comment:
i don't think we need this: NonZeroU8 and NonZeroI8 can both map to NonZeroU8 as a ULE type.

Reply:
Done